docs/get-started.md

```diff
@@ -46,6 +48,42 @@ NOTE: Make sure to check out the appropriate version. We recommend that you chec
     kubectl get deployments --namespace=nginx-example
     ```
 
+### (Optional) Edit configurations for different environments
```

Comment:
The title here is pretty generic, I'd prefer something more like "expose Minio outside the cluster" (to be able to run `ark logs` and `ark describe`)